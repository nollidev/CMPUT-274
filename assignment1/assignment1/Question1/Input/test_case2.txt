johndoe@gmail.com janesmith@domain.ca admin@spam.com alice123@yahoo.net bob@fakeemail.org support@pleasenotspam.com contact@trashmail.net user@therealtaylorswift.com test@valid.edu invalid@gmail.com