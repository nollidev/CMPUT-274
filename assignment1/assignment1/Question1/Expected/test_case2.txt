Valid
Valid
Forbidden
Valid
Forbidden
Forbidden
Forbidden
Forbidden
Valid
Valid
